docs/user-guide/admin-cli.md

```diff
@@ -653,17 +701,144 @@ enhancement tracked [here](https://github.com/vmware/docker-volume-vsphere/issue
 [root@localhost:~] /usr/lib/vmware/vmdkops/bin/vmdkops_admin.py policy rm --name=some-policy
 Successfully removed policy: some-policy
 ```
+
+## Config
+
+Creates, removes, moves and reports on status of config DB. Config DB keeps authentication information, and without initializing it no access control is supported. Also, configuring access control will fail, e.g.
```

Comment:
May be rename auth_db to config_db, auth_db seems to indicate authentication which isn't done here..